packages/rest/src/session/doc/IOptionsForAddConnProps.ts

```diff
@@ -43,4 +43,15 @@ export interface IOptionsForAddConnProps {
      * The default value is TOKEN_TYPE_JWT.
      */
     defaultTokenType?: SessConstants.TOKEN_TYPE_CHOICES;
+
+    /**
+     * Specifies the functionality that external applications will use for prompting.
+     * Activate this function if needed propoerties are to be prompted one by one
+     */
+    getValuesBack? (properties: string): any;
```

Comment:
Would it make sense for this to be singular (value instead of values, property instead of properties) since it fetches only a single value?
`getValueBack? (property: string): any;`

Reply:
changed.